packages/website/docs/sources.md

```diff
@@ -53,7 +53,9 @@ autocomplete({
           return [
             { label: 'Twitter', url: 'https://twitter.com' },
             { label: 'GitHub', url: 'https://github.com' },
-          ].filter((item) => item.toLowerCase().includes(query.toLowerCase()));
+          ].filter(({ label }) =>
+            label.toLowerCase().includes(query.toLowerCase())
+          );
```

Comment:
I find this clearer for docs. Wdyt?

```suggestion
          ].filter((item) =>
            item.label.toLowerCase().includes(query.toLowerCase())
          );
```

Reply:
This is mostly a matter of preference. I personally find destructured properties to be clearer most of the time.

All the code examples that document the Autocomplete APIs are using object destructuring (e.g., [here](https://deploy-preview-403--algolia-autocomplete.netlify.app/docs/sources#using-static-sources), [here](https://deploy-preview-403--algolia-autocomplete.netlify.app/docs/state#listening-to-state-changes), etc.) which, to me, brings clarity, makes the code terser, and avoids naming things that don't need to be named.

This example is consistent with the rest of the docs, and it doesn't remove any expressivity. You see the static array being filtered right here, so `label` doesn't come from nowhere.

Reply:
Fine by me then, no strong opinion. Figured it could have been easier to understand for new comers.

Reply:
I'd say let's be consistent, and we can update if we get feedback that code snippets are too hard to understand.